src/components/DnsPage.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div class="card-content">
+    <div class="content"
+          v-if="dnsRecords && !loadingDocCounter">
+      <rbz-table :columns="columns"
+                 :data="dnsRecords"
+                 :default-sort-column-index="1"
+                 :vertical-top="true"
+                 :show-filter-button="true">
+      </rbz-table>
+    </div>
+    <div class="content no-data-wrapper"
+         v-if="!selectedBranch || !dnsRecords">
```

Comment:
We can remove `!selectedBranch` if we're not branch dependant in this page

```suggestion
         v-if="!dnsRecords && loadingDocCounter">
```

Reply:
the api call is without branch.
However, I wonder if we should still include the branch to avoid, branch changing on reload the page.

Reply:
We can include the branch in the url, but we don't need to rely on it or read it inside this component